File: articles/atoms-molecules-and-the-structure-of-matter/c/bohr_energy_table.c
/*
Bohr Energy Table

This C workflow computes:

    E_n = -13.6 / n^2

for introductory hydrogen-like atomic levels.
*/

#include <stdio.h>

double bohr_energy_ev(int n) {
    return -13.6 / (double)(n * n);
}

int main(void) {
    printf("n,energy_ev\n");

    for (int n = 1; n <= 12; n++) {
        printf("%d,%.8f\n", n, bohr_energy_ev(n));
    }

    return 0;
}

File: articles/atoms-molecules-and-the-structure-of-matter/fortran/bohr_energy_table.f90
! Bohr Energy Table
!
! This Fortran workflow computes:
!
!   E_n = -13.6 / n^2
!
! for introductory hydrogen-like atomic levels.

program bohr_energy_table
  implicit none

  integer :: n
  real(8) :: energy_ev

  print *, "n,energy_ev"

  do n = 1, 12
     energy_ev = -13.6d0 / dble(n * n)
     print '(I4,A,F14.8)', n, ",", energy_ev
  end do

end program bohr_energy_table
